src/wp-includes/class-wp-theme-json.php

```diff
@@ -1934,6 +1934,15 @@ protected static function compute_style_properties( $styles, $settings = array()
 			return $declarations;
 		}
 
+		// The global settings can include dynamic data related to typography. We need evaluate it so that the cache is invalidated when it changes.
+		$args      = array( func_get_args(), wp_get_global_settings() ); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.Changed
+		$cache_key = 'compute_style_properties_' . md5( wp_json_encode( $args ) );
```

Comment:
Suggestion/nitpick: rename `$args` to `$cach_args` for readability.

```suggestion
		$cache_args      = array( func_get_args(), wp_get_global_settings() ); // phpcs:ignore PHPCompatibility.FunctionUse.ArgumentFunctionsReportCurrentValue.Changed
		$cache_key = 'compute_style_properties_' . md5( wp_json_encode( $cache_args ) );
```